packages/react-core/src/index.ts

```diff
@@ -1,2 +1,12 @@
 // main entry point = evm
 export * from "./evm";
+
+// export types from thirdweb-dev/sdk
+export type {
+  NFTMetadataInput,
+  NFT,
+  NFTMetadata,
+  NFTMetadataOrUri,
+} from "@thirdweb-dev/sdk";
+export type { CurrencyValue, TokenMetadata } from "@thirdweb-dev/sdk";
+export type { QueryAllParams } from "@thirdweb-dev/sdk";
```

Comment:
There are other types we may be missing here and that users may need. We may need to re-export more?

Some of these types: `NFTDrop`, `Token`, `SmartContract`. In SDK we do `export * from "./types";`, so there are a bunch of types also there